app/policies/presentation_policy.rb

```diff
@@ -6,4 +6,12 @@ def create?
   def new?
     create?
   end
+
+  def update?
+    person.present? && (person.admin? || person.presentations.include?(record))
```

Comment:
Wouldn't it be ok if `update?` returned nil since nil is falsey anyway, or does Pundit explode?

At the same time I see that it breaks the convention of the ? at the end of the method name....so i'm just curious.